samza-api/src/main/java/org/apache/samza/operators/windows/Windows.java

```diff
@@ -24,8 +24,11 @@
 import org.apache.samza.operators.triggers.Trigger;
 import org.apache.samza.operators.triggers.Triggers;
 import org.apache.samza.operators.windows.internal.WindowInternal;
+import org.apache.samza.operators.windows.internal.WindowType;
 
 import java.time.Duration;
+import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.Collection;
 import java.util.function.BiFunction;
 import java.util.function.Function;
```

Comment:
Exactly. That would have been my next question :) So let's remove it if there isn't sufficient clarity. Thanks!